admob/src/android/admob_android.cc

```diff
@@ -597,27 +586,85 @@ Java_com_google_firebase_admob_internal_cpp_BannerViewHelper_completeBannerViewF
 }
 
 extern "C" JNIEXPORT void JNICALL
-Java_com_google_firebase_admob_internal_cpp_BannerViewHelper_notifyStateChanged(  // NOLINT
-    JNIEnv* env, jclass clazz, jlong data_ptr, jint change_code) {
-  if (data_ptr == 0) return;  // test call only
+Java_com_google_firebase_admob_internal_cpp_BannerViewHelper_notifyBoundingBoxChanged(  // NOLINT
+    JNIEnv* env, jclass clazz, jlong data_ptr) {
+  firebase::admob::internal::BannerViewInternal* internal =
+      reinterpret_cast<firebase::admob::internal::BannerViewInternal*>(
+          data_ptr);
+  internal->NotifyListenerOfBoundingBoxChange(internal->bounding_box());
+}
+
+extern "C" JNIEXPORT void JNICALL
+Java_com_google_firebase_admob_internal_cpp_BannerViewHelper_notifyAdClicked(
+    JNIEnv* env, jclass clazz, jlong data_ptr) {
+  firebase::admob::internal::BannerViewInternal* internal =
+      reinterpret_cast<firebase::admob::internal::BannerViewInternal*>(
+          data_ptr);
+  internal->NotifyListenerAdClicked();
+}
 
+extern "C" JNIEXPORT void JNICALL
+Java_com_google_firebase_admob_internal_cpp_BannerViewHelper_notifyAdClosed(
```

Comment:
Same as my comment on a previous PR, consider shortening these names since they are passed into RegisterNatives anyway. (And you can put them in an anonymous namespace as well.)

Reply:
Yep, it'll be part of the RewardedAd PR for Android.